src/gvars.c

```diff
@@ -1481,39 +1481,20 @@ static StructGVarFunc GVarFuncs [] = {
     { "MakeReadWriteGVar", 1, "name",
       MakeReadWriteGVarHandler, "src/gap.c:MakeReadWriteGVar" },
```

Comment:
Quite some entries use the suffix `Handler` instead of the prefix `Func`. I'll ignore them for now.